### ISSUE TITLE
f strings break for Python versions below 3.6

### ISSUE THREAD

**Issue by yinleon** [SMAPPNYU/urlExpander#6]
Traceback:
```
packages/urlexpander/core/tweet_utils.py", line 101
    raise ValueError(f"{col} is not a column in the input dataframe")
                                                                   ^
SyntaxError: invalid syntax"
```

**Comment by yinleon** [comment]
I am going to remove f string got backwards compatibility.

**Comment by yinleon** [comment]
Fixed for version 0.34, see commit https://github.com/SMAPPNYU/urlExpander/commit/46f65a07a0b9c540e0dfdbbbe4540df10e76627f
